pkgs/development/python-modules/threadpoolctl/default.nix

```diff
@@ -20,7 +20,7 @@ buildPythonPackage rec {
     owner = "joblib";
     repo = pname;
     rev = version;
-    sha256 = "0sl6mp3b2gb0dvqkhnkmrp2g3r5c7clyyyxzq44xih6sw1pgx9df";
+    sha256 = "7UUjbX1IpXtUAgN48Db43Zr1u360UETSUnIHD6rQRLs=";
   };
 
   checkInputs = [ pytestCheckHook pytest-cov numpy scipy ];
```

Comment:
Can we patch out the usage of pytest-cov?

Reply:
it should be optional if we use pytestCheckHook

Reply:
Indeed, removed, thanks.